### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by igdiaysu** [senchabot-opensource/monorepo#221] [Ideas]
# yayın duyuru (streamcord)

- streamcord gibi duyuru
  - kategoriye ve kişiye göre özelleştirme > twitch title özelliği gibi

- Kategori değiştiğinde bunu da bildirebilir.
  - (default) etiket atmadan >> özelleştirilip etiket eklenebilir
  - Kategori değiştiğindeki kategori özgü mesaj ve yayının ilk açılışındaki kategoriye özgü mesaj farklı olabilir. >> bunun için üçüncü seviye child category gerekiyor data ağacına.

**Comment by igdiaysu** [comment]
# embeded mesaj (probot)

**Comment by igdiaysu** [comment]
# karşılama mesajı (...)

**Comment by rasitds** [comment]
Discord Bot Twitch kanali Stream Schedule Check

Senchabot kullanan yayincilarin Twitch'te scheduled takvimlerini check edip 1 saat onceden (veya ayarlanabilir sekilde) Discord serverlarinda etkinlikler olusturulabilsin. Bu ozellik yapilandirilmadan calismasin ve Discord botundan bir komut ile de gecici olarak disabled edilebilsin.

**Comment by rasitds** [comment]
Custom Voice Channels
Custom Commands

**Reply by igdiaysu** [reply]
> Temp voice channel

**Reply by igdiaysu** [reply]
### Custom Temp Voice Channels

#### Method - 1: _Ses kanalına tıklayarak oluşturma. (Örn. Astro bot(?))_

- Ayarların oluşan ses kanalının sohbetinde görsel olarak çıkması gerekiyor.
- + görsel olsa bile ses kanalının sohbetinde komutla ayarlanabilme olursa bu farklı kılan bir özellik olur.

#### Method - 2: _Chat command ile oluşturma_

- Ayarlar oluşturma esnasında yapılabilir.

`
!command-name [kisi-sayisi] [rols]
`

**Comment by rasitds** [comment]
Ses kanallarina girip hatirlatmada bulunmak, (orn. su icmeyi hatirlatmak, pomodoro hatirlatmak) icin bir text to speech servisi

**Reply by igdiaysu** [reply]
### TTS | Text To Speech Service

TTS servisinin ses kanallarına girmek yerine bir mesaja sağ tıkladığımızda gelen veya sunuculardaki dahili komutlardan olan `/tts [message]` kullanılabilir mi? Çünkü topluluklarda sunucu sahibi bu özelliği aktif ettiğinde kanallara gir çık olması üyeleri rahatsız edebilir. Bunun yerine user özgü yapılabilir mi? Bu olmuyorsa bile ses kanalında kişi varsa belli aralıklarla TTS yerine, ses kanalı chatine o kişiler için hatırlatma geçebilir.

**Reply by rasitds** [reply]
Bu konuda arastirma yapalim, Discord'da tts ozelliginin oldugunu unutmusum
Bu secenek bot tarafindan kullanilmaya uygun oldugunda bu servise ihtiyacimiz kalmayabilir

**Comment by rasitds** [comment]
YouTube kanali video post duyurulari

**Comment by igdiaysu** [comment]
### Duyuru Mesajı & Etkinlik Oluşturma

1. veri girdisi (Twitch: kullanıcı yayına başladı) sonrası bağımlı aksiyonlar +++
> streamcord'u kullanmaya devam edenler için.
> Senchabot'a duyuru mesajı attıranlarda kullanabilir mantıken.
- _act-1:_ x chatine duyuru mesajı atılacak
- senchabot duyuru mesajını okuyacak:
- _act-2:_ senchabot okuduğu mesaja göre etkinlik oluşturacak

~~2. veri girdisi (Twitch: kullanıcı yayına başladı) sonrası ayrı iki aksiyon~~
~~stream duyurusunu da senchabot'a attıranlar için~~
~~- _act-1:_ x chatine duyuru mesajı atılacak~~
~~- _act-2:_ etkinlik oluşturulacak~~
